dart/welcome_email/main.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:convert';
+import 'dart:io';
+import 'package:http/http.dart' as http;
+
+Future<void> start(final req, final res) async {
+  final domain = req.env['MAILGUN_DOMAIN'];
+  final mailgun = MailgunMailer(
+    apiKey: req.env['MAILGUN_API_KEY']!,
+    domain: domain ?? '',
+  );
+
+  // Get the name and email of the newly created user from the payload
+  final payload = jsonDecode(req.payload!);
+  final name = payload['name'];
+  final email = payload['email'];
```

Comment:
We might want to introduce input validation here? Eldad's call, just suggesting.

Reply:
Let's add input validation, we want to have good practices in the official examples.

Reply:
I've added regex email validation and a check to make sure the name is in the payload